chain-test/src/test/scala/org/bitcoins/chain/blockchain/ChainHandlerTest.scala

```diff
@@ -31,4 +33,38 @@ class ChainHandlerTest extends ChainUnitTest {
       foundHeaderF.map(found => assert(found.get == newValidHeader))
   }
 
+  it must "be able to process and fetch real headers from mainnet" taggedAs FixtureTag.PopulatedChainHandler inFixtured {
+    case ChainFixture.PopulatedChainHandler(chainHandler) =>
```

Comment:
So if I understand this correctly, this presumes a already populated `BlockHeaderDAO`, i would like to popuplate the `BlockHeaderDAO` in this test case by calling `ChainHandler.processHeader()`